src/domain/auth/components/login/Login.tsx

```diff
@@ -61,6 +62,7 @@ function Login(props: Props) {
                     onClick={authenticate}
                     variant="supplementary"
                     className={styles.button}
+                    iconRight={null}
```

Comment:
Newer versions of HDS require `iconLeft`/`iconRight` for supplementary buttons. For now I avoided design changes by using `null`.